docs/source/schemas/streams.rst

```diff
@@ -32,22 +32,29 @@ The minimal parameters required to define a stream store are shown below:
 
 Additional options for the builder include:
 
-- ``hashFirstRowComponent``
-- ``isAppendHeavyAndReadLight``
-- ``compressBlocksInDb``
-- ``compressStreamInClient``: transparently decompresses and compresses the 
-  stream via the LZ4 algorithm upon reads and writes, respectively. 
-  Compression is performed client side before any network communication to 
-  the underlying database.
-- ``inMemoryThreshold(int inMemoryThreshold)``: specifies the largest size
-  object (in bytes) which AtlasDB will cache in memory in order to boost
-  retrieval performance.
-  
-For an example of streams in use, see `ProfileSchema`_ and `ProfileStore`_.
+.. list-table::
+    :widths: 20 80
+    :header-rows: 1
+
+    *    - Option
+         - Description
+
+    *    - ``hashFirstRowComponent``
+         - Hashes the ``id`` component of the table.
 
-.. _ProfileSchema: ../../../examples/profile-client/src/main/java/com/palantir/example/profile/schema/ProfileSchema.java
-.. _ProfileStore: ../../../examples/profile-client/src/main/java/com/palantir/example/profile/ProfileStore.java
+    *    - ``isAppendHeavyAndReadLight``
+         - ``false`` by default. If set to ``true``, some optimisations will be made in Cassandra KVS assuming that these streams will be changed often but not read often.
+
+    *    - ``compressStreamInClient``
+         - Transparently decompresses and compresses the stream via the LZ4 algorithm upon reads and writes, respectively. Compression is performed client side before any network communication to the underlying database.
+
+    *   - ``inMemoryThreshold``
+        - Specifies the largest size object (in bytes) which AtlasDB will cache in memory in order to boost retrieval performance.
+  
+For an example of streams in use, see the ``user_profile`` table and ``user_photos`` stream store in `ProfileSchema`_, and the ``updateImage`` method in `ProfileStore`_.
 
+.. _ProfileSchema: https://github.com/palantir/atlasdb/blob/cd4f33dfcaa95acb90374f698158a4aae8c28945/examples/profile-client/src/main/java/com/palantir/example/profile/schema/ProfileSchema.java
+.. _ProfileStore: https://github.com/palantir/atlasdb/blob/cd4f33dfcaa95acb90374f698158a4aae8c28945/examples/profile-client/src/main/java/com/palantir/example/profile/ProfileStore.java
```

Comment:
You can add `#L92` to this link, to link the viewer directly to the relevant part of the file.